### PR TITLE
feat(replays): Update feature flag for replayer plugin

### PR DESCRIPTION
**Pull request by c298lee**
Update the replayer plugin feature flag to `session-replay-enable-canvas-replayer`

### CODE REVIEW

**Review by ryan953 — APPROVED** [review]

File: static/app/components/replays/replayContext.tsx [hunk 1]
```diff
@@ -373,7 +373,7 @@ export function Provider({
           lineWidth: 2,
           strokeStyle: theme.purple200,
         },
-        plugins: organization.features.includes('session-replay-enable-canvas')
```
Comment (ryan953) [inline static/app/components/replays/replayContext.tsx:376]: Should we be going around and removing this flag if it's old/unused now?

File: static/app/components/replays/replayContext.tsx [hunk 2]
```diff
@@ -373,7 +373,7 @@ export function Provider({
...
+        plugins: organization.features.includes('session-replay-enable-canvas-replayer')
```
Comment (ryan953) [inline static/app/components/replays/replayContext.tsx:376]: Also, this is added to those 2 python files right? That should be a separate PR so it deploys first

Comment (c298lee) [reply static/app/components/replays/replayContext.tsx:376]: yup I made a seperate PR for it!

File: static/app/components/replays/replayContext.tsx (hunk 1 above)
Comment (c298lee) [reply static/app/components/replays/replayContext.tsx:376]: This still exists, it's used for recording canvas! We decided to add a new feature flag for replaying canvas.

**Review by billyvg — APPROVED** [review]
